angular/core/components/group_page/group_welcome_modal/group_welcome_modal.coffee

```diff
@@ -1,8 +1,25 @@
 angular.module('loomioApp').factory 'GroupWelcomeModal', ->
   templateUrl: 'generated/components/group_page/group_welcome_modal/group_welcome_modal.html'
   size: 'group-welcome-modal'
-  controller: ($scope, AppConfig, UserHelpService) ->
-    $scope.showVideo = AppConfig.loadVideos
+  controller: ($scope, group, AppConfig, UserHelpService, AbilityService) ->
+    $scope.group = group
+    $scope.showVideo = true
```

Comment:
Why not take out `showVideo` entirely and have the videos always shown in the template? 💛